setup/freebsd/ansible-vars.yaml

```diff
@@ -4,8 +4,9 @@ init_script_path: /usr/local/etc/rc.d/jenkins
 ssh_users:
   - rvagg
   - jbergstroem
+  - joaocgreis
 packages:
-  - openjdk-jre
+  - openjdk
```

Comment:
I've seen discrepancies between the different package sources of openjdk (the stuff on DO is slightly different from what I've seen at voxer). OpenJDK had less dependencies for some reason.